clustered/common/src/main/java/org/ehcache/clustered/common/ServerStoreCompatibility.java

```diff
@@ -76,6 +77,26 @@ public void verify(ServerStoreConfiguration serverConfiguration, ServerStoreConf
     if (!isCompatible) {
       throw new ClusteredStoreValidationException(sb.toString());
     }
+
+  }
+
+  private static boolean comparePoolAllocationType(StringBuilder sb, PoolAllocation serverPoolAllocation, PoolAllocation clientPoolAllocation) {
+    if ((serverPoolAllocation == null && clientPoolAllocation == null)
```

Comment:
What does it mean for both to be null?  Surely this method should never be called with both being null?